.github/workflows/runners.yml

```diff
@@ -40,11 +40,28 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
-      - name: Setup Docker Environment
+      - name: Lint shell
+        run: |
+           make -C runner lint-scripts
```

Comment:
Hey @CASABECI! Thanks for a lot for your contribution.
I finally managed to revisit this PR and have some time to integrate your work into the project,
and realized that we now have an exclusion rulle for `runner/Makefile` so if we added this `lint-scripts` target to the this file, it won't get triggered when the Makefile has changed.
Probably we'd better roll a new workflow dedicated for running spellcheck... Let me try.